src/Swarm/TUI/Controller.hs

```diff
@@ -836,7 +836,7 @@ handleREPLEventTyping = \case
             -- `reprogram` at runtime.  See the discussion at
             -- https://github.com/swarm-game/swarm/pull/827 for more
             -- details.
-            . (gameState . baseRobot . robotContext . defReqs .~ reqCtx)
+            . (gameState . baseRobot . robotContext . defReqs <>~ reqCtx)
```

Comment:
> We simply dump the entire `reqCtx` into the robot's context

Should the comment say _add_ instead of _dump_? :slightly_smiling_face: To me, dumping could mean overwrite which I guess it did.